ansible-contrib/roles/master/handlers/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: reload systemd
+  command: systemctl --system daemon-reload
+  notify:
+    - restart daemons
+  when: not (ansible_distribution == "Ubuntu" and ansible_distribution_major_version|int < 15) or (is_coreos)
```

Comment:
Does this mean "when: is_systemd"?  Can we declare that somewhere (as we do today)?